overlord/snapstate/snapmgr.go

```diff
@@ -390,39 +390,37 @@ func refreshScheduleNoWeekdays(rs []*timeutil.Schedule) error {
 	return nil
 }
 
-func (m *SnapManager) checkRefreshSchedule() ([]*timeutil.Schedule, error) {
+func resetRefreshScheduleToDefault(st *state.State) (ts []*timeutil.Schedule, scheduleStr string) {
+	refreshSchedule, err := timeutil.ParseSchedule(defaultRefreshSchedule)
+	if err != nil {
+		panic(fmt.Sprintf("defaultRefreshSchedule cannot be parsed: %s", err))
+	}
+	tr := config.NewTransaction(st)
+	tr.Set("core", "refresh.schedule", defaultRefreshSchedule)
+	tr.Commit()
+
+	return refreshSchedule, defaultRefreshSchedule
+}
+
+func (m *SnapManager) refreshScheduleWithDefaultsFallback() (ts []*timeutil.Schedule, scheduleAsStr string, err error) {
 	refreshScheduleStr := defaultRefreshSchedule
 
 	tr := config.NewTransaction(m.state)
-	err := tr.Get("core", "refresh.schedule", &refreshScheduleStr)
+	err = tr.Get("core", "refresh.schedule", &refreshScheduleStr)
 	if err != nil && !config.IsNoOption(err) {
-		return nil, err
+		return nil, "", err
```

Comment:
Just `return` is sufficient if returned arguments are named

Reply:
we tend not to use bare return, we also have a checker in our static tests to make sure we really don't do that in large functions